internal/command/darwin.go

```diff
@@ -67,7 +67,7 @@ func (cmd *darwin) Parse(args []string) error {
 	if runtime.GOOS == darwinOS {
 		flagSet.BoolVar(&cmd.localBuild, "local", true, "If set uses the fyne CLI tool installed on the host in place of the docker images")
 	} else {
-		flagSet.StringVar(&flags.MacOSXSDKPath, "macosx-sdk-path", "unset", "Path to macOS SDK [required]")
+		flagSet.StringVar(&flags.MacOSXSDKPath, "macosx-sdk-path", "unset", "Path to macOS SDK (setting it to 'ignore' will skip the in container sdk check) [required]")
```

Comment:
I can't quite follow "'ignore' will skip the in container sdk check" - I wonder if "ignore" is more about the implementation rather than the intent.
Would it not mean more to the developer if it was `(set to 'bundled' if the SDK is included in your image)`.
The original reads as though it is possible to have no SDK (i.e. ignore the error of none present)

Reply:
`bundled` would work indeed. Will update.